src/main/java/org/akaza/openclinica/controller/ChangeCRFVersionController.java

```diff
@@ -583,10 +585,10 @@ public ModelMap changeCRFVersionAction(HttpServletRequest request, HttpServletRe
             seb.setUpdatedDate(new Date());
 
             status_before_update = auditDao.findLastStatus("study_event", seb.getId(), "8");
-            if (status_before_update != null && status_before_update.length() == 1) {
-                int status = Integer.parseInt(status_before_update);
-                eventStatus = SubjectEventStatus.get(status);
-                seb.setSubjectEventStatus(eventStatus);
+            status_before_update_new = auditDao.findLastStatus("study_event", seb.getId(), "SIGNED");
```

Comment:
can you reference the enum value here instead of a string constant?

Reply:
done